app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def avatar_url(user)
+    gravatar_id = Digest::MD5::hexdigest(user.email).downcase
+    "http://gravatar.com/avatar/#{gravatar_id}.png?s=80"
+  end
```

Comment:
I would make sure your desired hashing algorithm is called with the right argument. Right now you're calling `downcase` on a hash, is that what you want? A test would help with that ;)